connectrum/client.py

```diff
@@ -72,22 +82,25 @@ def close(self):
         hostname, port, use_ssl = server_info.get_port(proto_code)
 
         if use_tor:
-            # Connect via Tor proxy proxy, assumed to be on localhost:9050
-            # unless a tuple is given with another host/port combo.
-            try:
-                socks_host, socks_port = use_tor
-            except TypeError:
-                socks_host, socks_port = 'localhost', 9150
+            if have_aiosocks:
+                # Connect via Tor proxy proxy, assumed to be on localhost:9050
+                # unless a tuple is given with another host/port combo.
+                try:
+                    socks_host, socks_port = use_tor
+                except TypeError:
+                    socks_host, socks_port = 'localhost', 9150
```

Comment:
You mean that the port number in the comment is not the same as in the code? I didn't change that. It already like that in the code.